### PR TITLE
Reverse A/V/L QC scores

### PR DESCRIPTION
**Pull request by tashrifbillah**

### CODE REVIEW

File: combine_avlqc.py [hunk 1]
```diff
@@ -49,9 +80,11 @@ def concat_site_csv(data_root,output_root,center_name):
         open_only.dropna(subset=["overall_db","num_inaudible"],how='all',inplace=True) # remove records with only video for these purposes, focus on audio
         open_only.reset_index(drop=True,inplace=True)
         open_only["inaudible_per_word"] = [x/(a + b + c) if not np.isnan(x) else np.nan for x,a,b,c in zip(open_only["num_inaudible"].tolist(),open_only["num_words_S1"].tolist(),open_only["num_words_S2"].tolist(),open_only["num_words_S3"].tolist())]
-        # 0 awaiting transcription, 1 excellent (<1% inaud), 2 good (<5% inaud), 3 fair (<20% inaud), 4 usable (>20% inaud but transcript available), 5 bad (db < 40 so not sent for transcription)
-        # note that interviews missing from audio QC due to SOP violations or other issues will not be reflected here at all! these counts relate only to interviews that were able to be processed by QC
-        open_only["audio_quality_category"] = [0 if np.isnan(x) and y > 40 else (5 if np.isnan(x) else (1 if x < 0.01 else (2 if x < 0.05 else (3 if x < 0.2 else 4)))) for x,y in zip(open_only["inaudible_per_word"].tolist(),open_only["overall_db"].tolist())]
+
+        
+        open_only["audio_quality_category"] = get_score(
```
Comment (tashrifbillah) [inline combine_avlqc.py:85]: Scoring is now functionalized, so future modification can be done in one place.

File: combine_avlqc.py [hunk 2]
```diff
@@ -1,12 +1,43 @@
...
+        elif x < 0.05:
+            _score=4
+        elif x < 0.2:
+            _score=3
+        else:
+            _score=2
+        
+        score.append(_score)
+        
+    return score
```
Comment (tashrifbillah) [inline combine_avlqc.py:38]:
```python
psychs_only["audio_quality_category"] = [0 if np.isnan(x) and y > 40 else (5 if np.isnan(x) else (1 if x < 0.01 else (2 if x < 0.05 else (3 if x < 0.2 else 4)))) for x,y in zip(psychs_only["inaudible_per_word"].tolist(),psychs_only["overall_db"].tolist())]
```

This was Michaela's original code. I found it to be complex for our successors to understand. So I simplified it.

Comment (tashrifbillah) [reply combine_avlqc.py:38]: Also, as the goal of this PR states, I reversed the scores. So more number means better quality now and vice-versa. On the other hand, score `0` has been kept the same.

**Comment by ennisthemennis** [comment]
Looks fine to me, thanks!

**Pull request merged**